src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -60,6 +62,7 @@ const CreateOrEditQuizContainer: React.FC<QuizContentProps> = ({
   const { id = '' } = useParams()
   const { handleErrorAlert, handleAlert } = useSnackbarAlert()
   const [isCreationOpen, setIsCreationOpen] = useState<boolean>(false)
+  const modalRef = useRef<CreateOrEditQuizQuestionRef>(null)
```

Comment:
Use React `ComponentRef` (import it from React) instead of manual typing
```suggestion
  const modalRef = useRef<ComponentRef<typeof CreateOrEditQuizQuestion> | null>(null)
```